modules/appeals_api/catalog-info.yaml

```diff
@@ -1,11 +1,21 @@
 apiVersion: backstage.io/v1alpha1
-kind: Location
+kind: API
 metadata:
-  name: appeals-apis
```

Comment:
This file seems to represent all of the `appeals_api` module, including both the Appeals Status API and the Decision Reviews API. That seems wrong, since there's another file (below) that just represents the Appeals Status API, so I changed this file to represent Decision Reviews